README.md

```diff
@@ -114,18 +146,65 @@ This is the initial value of the form checkbox.
 Custom function to translate or modify input label.
 
 ### trueValue {string} - optional
-Value for checked checkbox. Default: `yes`.
+Value for checked checkbox. Default: `true`.
```

Comment:
default true? nebude lepšie `0` a `1`? tak tomu porozumie aj laravel aj adonis

Reply:
Ok jasne mozeme to zmenit na 0 a 1

Reply:
možno by bolo fajn mať tam true a false value keď to je checkbox

Reply:
preto tam je to trueValue, falseValue aby si si mohol povedat co chces poslat kedy